charts/retool/templates/deployment_workflows_worker.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.workflows.enabled }}
+{{- if include "retool.workflows.enabled" . }}
```

Comment:
whoa, this works? how does it coerce to boolean, output empty vs nonempty?

Reply:
yup, exactly! https://helm.sh/docs/chart_template_guide/control_structures/#:~:text=A%20pipeline%20is%20evaluated%20as,an%20empty%20string